news/2019-08-10-new-featured-artist-undead-corporation.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "New Featured Artist: UNDEAD CORPORATION"
+date: 2019-08-10 12:00:00 +0000
+---
+
+The Empress beckons. [**UNDEAD CORPORATION**](https://osu.ppy.sh/beatmaps/artists/45) shambles into the Featured Artist lineup with over a hundred tracks to choose from!
+
+![](https://assets.ppy.sh/artists/45/header.jpg)
+
+You know them, you love them. From timeless classics like *Everything will freeze* and *The Empress*, **UNDEAD CORPORATION** have held a dear spot in the osu! community's heart for the better part of a decade.
+
+And now, we are delighted to announce once and for all that it is finally official!
+
+**111** metal masterpieces await over at [their Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/45) for you to sink your teeth into, with many of these covering the considerable library of already ranked beatmaps that we're all familiar with.
+
+We've also managed to snag the full ensemble of their two original albums - *O.D* and *NO ANTIDOTE*. Be sure to check them out!
+
+In case you've forgotten, check out the samples below:
+
+### Everything will freeze
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1075.mp3" type="audio/mpeg">
+</audio>
+
+### The Empress
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1084.mp3" type="audio/mpeg">
+</audio>
+
+### Kami Kakushi
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1075.mp3" type="audio/mpeg">
+</audio>
+
+### Forgive your insanity
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1108.mp3" type="audio/mpeg">
+</audio>
+
+### Slash your mind
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1130.mp3" type="audio/mpeg">
+</audio>
+
+### No Antidote
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1168.mp3" type="audio/mpeg">
+</audio>
+
+### Gone With The Blast
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/45/previews/1177.mp3" type="audio/mpeg">
+</audio>
+
+---------
+
+As usual, all of these amazing tracks and a **HUGE** amount more are available right now over at [**UNDEAD CORPORATION**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/45), pre-timed for your convenience. 100% free for use in mapping, so get out there and get creating with them!
```

Comment:
```suggestion
As usual, all of these amazing tracks and a **HUGE** amount more are available right now over at [**UNDEAD CORPORATION**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/45), pre-timed for your convenience. These are 100% free for use in mapping, so get out there and get creating with them!
```
since you used a full stop.